### PR TITLE
ecobee has now enabled 2FA (SMS)

### PR DESCRIPTION
**Pull request by LeeBrotherston**
ecobee has now enabled 2FA for all customers.

### CODE REVIEW

File: _data/iot.yml [hunk 1]
```diff
@@ -14,6 +14,9 @@ websites:
     twitter: ecobee
     facebook: ecobee
```
Comment (phallobst) [inline _data/iot.yml]:
```suggestion
```

File: _data/iot.yml [hunk 2]
```diff
@@ -14,6 +14,9 @@ websites:
...
     img: ecobee.png
+    tfa:
+      - sms
+    doc: https://support.ecobee.com/hc/en-us/articles/360040957212-Two-Factor-Authentication-2FA-
```
Comment (phallobst) [inline _data/iot.yml]:
```suggestion
    doc: https://support.ecobee.com/hc/en-us/articles/360040957212
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: _data/iot.yml (hunk 1 above)
Comment (LeeBrotherston) [reply _data/iot.yml]: Those are both the correct accounts.

Comment (phallobst) [reply _data/iot.yml]: Yes, but they are not used/needed if some tfa method is present (see the failed Jekyll check).

Comment (LeeBrotherston) [reply _data/iot.yml]: Aaaah gotcha.... That makes sense.  Sorry... lack of coffee fail!

**Review by phallobst — APPROVED** [review]

**Review by jamcat22 — APPROVED** [review]

**Pull request merged**